021-IdentityForApps/Coach/README.md

```diff
@@ -0,0 +1,59 @@
+# Identity for Developers
+
+## Introduction
+
+Welcome to the coach's guide for the Identity for Developers Hack. Here you will find links to specific guidance for coaches for each of the challenges.
```

Comment:
I might have missed it but to explain to the coaches anywhere what the files in Coach/Solutions/Apps are for?

Also just double checking, these aren't identical copies of the apps in Student/Resources are they? They seem to be full projects. If they have completed code, it's better to just put the CHANGED files with the solution code in Coach/Solutions, not a full copy of the app. It's understood that you'd use the student app and overlay the solution on top.

Reply:
This is a duplicate - removed now